.github/workflows/build_and_test.yml

```diff
@@ -43,8 +43,11 @@ jobs:
         echo "$GITHUB_WORKSPACE/openmpi-${OMPI_VERSION}/installed/bin" \
           >> $GITHUB_PATH
 
+    - name: Check Debug build
+      run: FC=mpif90 cmake -S . -B build -DCMAKE_BUILD_TYPE=Debug
```

Comment:
I think this is just configuring cmake, but not running the compilation itself. If github actions are run in order of the yaml file the release configuration will be used for the build itself.